gateway-export-plugin/src/test/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/loader/PolicyLoaderTest.java

```diff
@@ -6,136 +6,117 @@
 
 package com.ca.apim.gateway.cagatewayexport.tasks.explode.loader;
 
-import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.PolicyType;
-import com.ca.apim.gateway.cagatewayconfig.util.entity.EntityTypes;
-import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentTools;
-import com.ca.apim.gateway.cagatewayexport.tasks.explode.bundle.entity.PolicyEntity;
-import com.google.common.collect.ImmutableMap;
-import org.junit.jupiter.api.Test;
-import org.w3c.dom.Document;
-import org.w3c.dom.Element;
-
-import java.util.Map;
-import java.util.UUID;
-
-import static com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.PolicyType.*;
-import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BuilderUtils.buildAndAppendPropertiesElement;
-import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BundleElementNames.*;
-import static com.ca.apim.gateway.cagatewayconfig.util.properties.PropertyConstants.PROPERTY_TAG;
-import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.*;
-import static org.junit.jupiter.api.Assertions.*;
-
 class PolicyLoaderTest {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add some tests to this class. ([squid:S2187](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2187))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=122&issues=AWblzesZSgN4uCclhZKa&open=AWblzesZSgN4uCclhZKa)

[ISSUE_KEY]: AWblzesZSgN4uCclhZKa